coalahtml/_coalahtml/tests/test_projects/simple/test.py

```diff
@@ -1,13 +1,12 @@
 import sys
-entry_point=sys.argv[1]
+entry_point = sys.argv[1]
```

Comment:
Ah this file doesn't even exists anymore. I just had to fix these PEP8 just to mute the warnings from Gitmate. :pensive: